README.md

```diff
@@ -39,18 +39,20 @@ configuration SYSTEM_MSFeedSync
     Import-DscResource -ModuleName WindowsDefenderDsc
     node localhost
     {
-        ProcessMitigation SYSTEM
+        ProcessMitigation Firefox
         {
-            MitigationTarget = 'SYSTEM'
-            Enable           = 'DEP', 'SEHOP'
-            Disable          = 'TerminateOnError'
+            MitigationTarget = 'firefox.exe'
```

Comment:
Can we line up the equal signs to make it look pretty?

Reply:
Done